pkg/gadgettracermanager/k8s/podinformer.go

```diff
@@ -101,7 +103,23 @@ func NewPodInformer(node string, createdPodChan chan *v1.Pod, deletedPodChan cha
 }
 
 func (p *PodInformer) Stop() {
+	// tell all workers to end
 	close(p.stop)
+
+	// wait for workers to end before closing channels to avoid
+	// writing to closed channels
+	p.wg.Wait()
+
+	close(p.createdPodChan)
+	close(p.deletedPodChan)
+}
+
+func (p *PodInformer) CreatedChan() chan *v1.Pod {
```

Comment:
We could specify a channel directions to increase type safety:

```suggestion
func (p *PodInformer) CreatedChan() <-chan *v1.Pod {
```

Reply:
TIL! Thanks!